### ISSUE TITLE
Forms Dashboard update - Debt Resolution Streamlined Waiver 5655 FSR

### ISSUE THREAD

**Issue by Tom-Davis** [department-of-veterans-affairs/va.gov-team#62443]
## What this form is for

<details>
<summary>More Information</summary>
<div>
    <br>
<p>Use this template to request Google Analytics implementation or QA for your product. Please fill out all brackets.</p>
    
> _Please see the following information for implementation timeline:_
> New Teams: Implementation will usually be completed in 4 sprints
> 
> - Sprint 1: Discovery/Orientation
> - Sprint 2: FE Implementation
> - Sprint 3: Tagging Implementation
> - Sprint 4: QA
> 
> All other teams: Implementation will usually be completed in 2 sprints
> 
> - Sprint 1: Discovery and FE Implementation
> - Sprint 2: Tagging Implementation and QA

</div>
</details>

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [X] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**

    - Debt Resolution
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - New URLs:
    - /monetary-asset-cash
    - /skip-questions-explainer
    
- **When is the launch date?**
    - 08/17/2023

- **What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_
        
    - Page visitors (total count and unique), is this part of the standard Design System Components?

    - [If this is a funnel, insert each step's URL - pages visible depending on the applicant's answers should be marked accordingly]
    
    - [Insert other interactions - please review the [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) and [GA Data Dictionary](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Google-Analytics-Data-Dictionary.1810464912.html)  to make sure these interactions aren't already tracked]

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - [Insert test user]

- **Any additional comments**

    - [Insert comments]

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by michelle-dooley** [comment]
@Tom-Davis so you are adding those above urls to the debt resolution form funnel?

**Comment by Tom-Davis** [comment]
> @Tom-Davis so you are adding those above urls to the debt resolution form funnel?

@michelle-dooley - yes, those 2 are being added to the funnel.

**Comment by michelle-dooley** [comment]
@Tom-Davis - [here](https://docs.google.com/spreadsheets/d/1k47yvsL0SLhrBrXme_Fp8xVsSLGfW1R1i-mrgNR2jYg/edit#gid=1220834450) are the urls we are currently using.  Can you confirm they are still correct and add the new ones where they belong in the flow.

**Comment by Tom-Davis** [comment]
@michelle-dooley let me grab the developers to help confirm.

@amponce @OptionSelect @Dr-Pongo can you help with the URL verification?

> @Tom-Davis - [here](https://docs.google.com/spreadsheets/d/1k47yvsL0SLhrBrXme_Fp8xVsSLGfW1R1i-mrgNR2jYg/edit#gid=1220834450) are the urls we are currently using. Can you confirm they are still correct and add the new ones where they belong in the flow.

**Comment by michelle-dooley** [comment]
@Tom-Davis @amponce @OptionSelect @Dr-Pongo - hey there just checking in on the status of the updated url list for this request. We need that before we can begin this work.

**Comment by Dr-Pongo** [comment]
Hey @michelle-dooley These are getting added to the FSR funnel, not the debt resolution funnel. We updated the FSR ones recently, I'm going to try and find it and link it here!

**Comment by michelle-dooley** [comment]
@Dr-Pongo - ok guess I was confused by the name of this ticket Debt Resolution.  So you are add the 2 urls above to the flow you just updated in that ticket?  Where in the flow do they go?

**Comment by Dr-Pongo** [comment]
@michelle-dooley 
Yeah! I can make a new tab on that google sheet specifically for this update if that helps? The order changed a bit for some of the other URLs as well. 

If we reuse the enhanced we can do the following:
- `/monetary-asset-cash` is optional can go before `/monetary-asset-checklist` (just after `/dependent-ages`). 
- `/skip-questions-explainer` is optional and can come immediately after `/monetary-asset-cash` or `/add-other-expense`

**Comment by michelle-dooley** [comment]
@Dr-Pongo - I added monetary-asset-cash to the [funnel worksheet V3 tab](https://docs.google.com/spreadsheets/d/1ZlwjHdCAZZLp2k7QRX0Bf0xsEBr98W_UYUW_rjW7Hlk/edit#gid=1512782164) please confirm I put in correct url and placement.

I am unclear on the skip-questions-explainer - are you saying that same url could follow monetary-asset-cash AND add-other-expense? If so we would have the same url with 2 different step numbers and the system is not smart enough to know the difference b/c it is all based on url.  So should we just put after one or the other or not at all? Or if needs to be in both places then there needs to be 2 different urls.

**Comment by michelle-dooley** [comment]
@Dr-Pongo - you just won't know if they got through from the /monetary-asset-cash page or the /add-other-expense page. Not sure how important that is to know or if you just need to know how many people went to that link regardless of source?

**Comment by michelle-dooley** [comment]
Sprint update: The QA work to complete this ticket will be completed in the next sprint.

**Comment by HeatherRienks** [comment]
@michelle-dooley - yay, thank you! What are your sprint dates and we will share the news with the team?!

**Comment by michelle-dooley** [comment]
@HeatherRienks - this work has been completed, please see V3 in funnel filter drop down. I noticed that I do not see anything coming through for /monetary-asset-cash url not sure if maybe that has not launched yet, looked in GA and don't see it there either. 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBKzhnQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eec76a4642e9de23dce029bb1c975a90bd0c4a5e/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0FnQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2fb8b0447ee64011269e753fba9cfb63e654da36/image.png)

**Comment by HeatherRienks** [comment]
@michelle-dooley - thanks for your help and the update!

@Tom-Davis @denisecoveyduc @Dr-Pongo - FYI, and any feedback on Michelle's comment? Thanks!
